darts/tests/explainability/test_shap_explainer.py

```diff
@@ -552,3 +569,51 @@ def test_plot(self):
             target_component="power",
         )
         self.assertTrue(isinstance(fplot, shap.plots._force.BaseVisualizer))
+
+    def test_feature_values_validity(self):
+        model = LightGBMModel(
+            lags=4,
+            output_chunk_length=1,
+        )
+        model.fit(
+            series=self.target_ts,
+        )
+        shap_explain = ShapExplainer(model)
+        explanation_results = shap_explain.explain()
+        df = pd.merge(
+            self.target_ts.pd_dataframe(),
```

Comment:
We could try to make this more robust if we also test `explain()` or `get_feature_values()` output against the actual shap output.

Reply:
Done in separate unit test